### ISSUE TITLE
install.sh doesn't install all cron jobs

### ISSUE THREAD

**Issue by nhandler** [matfra/isitfoggy.today#5]
Currently, `install.sh` only installs the `timelapse` cron job. It would be nice if it could also optionally install the other cron jobs that ship with this repository.

**Comment by matfra** [comment]
Yep, the DNS thing should definitely be added. For the ssl renewal, certbot already install a cronjob by default,but it fails because it needs to listen to 443 bit nginx is taking it. I'm wondering if there is a better solution than my cronjob
